book/html.md

```diff
@@ -370,6 +370,8 @@ looks something like this:
  <html lang="en-US" xml:lang="en-US">
    <head>
      <meta charset="utf-8" /="">
+       <link rel="prefetch" ...>
+         <link rel="prefetch" ...>
```

Comment:
Next paragraph refers to things being deeply indented; we need more lines to actually demonstrate that. The `...` fills in for some actually very long tag names.